src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    statusCheck: (state) => ({ ...state, status: 'Under construction' }),
```

Comment:
**OPTIONAL** - There would not be a change in the state since your initial state already has the value of `status=Under construction`. According to the project requirements you should `Create an initialState with a state value categories that contains an empty array` or you can set the value of status in the initial state to an empty string. Please fix this 😄

Reply:
@mckent05 I fixed it, thanks